apps/examples/smartfs_powercut/smartfs_powercut_main.c

```diff
@@ -171,98 +212,151 @@ static int create_file(char *path, ssize_t bufsize, file_type_t type)
 	return ret;
 }
 
-static int verify_random_file(char *path, int bufsize, int *index)
+static int create_sample_text_file(void)
+{
+	int ret;
+	int fd_sample;
+
+	fd_sample = open(SAMPLE_TXT_FILE, O_CREAT | O_WROK);
+	if (fd_sample < 0) {
+		printf("Unable to create sample file: %s, errno: %d\n", SAMPLE_TXT_FILE, errno);
+		ret = ERROR;
+		goto error_with_fd_sample;
+	}
+
+	for (int i = 0; i < TEST_WRITECOUNT; i++) {
+		memset(g_buf.data, 0xff, TEST_LARGE_BUFSIZE);
+		make_random_chars(g_buf.data, TEST_LARGE_BUFSIZE);
+
+		ret = write(fd_sample, (void *)g_buf.data, TEST_LARGE_BUFSIZE);
+		if (ret != TEST_LARGE_BUFSIZE) {
+			printf("Unable to write to file: %s, ret: %d\n", SAMPLE_TXT_FILE, ret);
+			ret = ERROR;
+			goto error_with_fd_sample;
+		}
+	}
+
+	ret = OK;
+error_with_fd_sample:
+	close(fd_sample);
+	return ret;
+}
+
+static int verify_file(char *path, int bufsize, int *index, file_type_t type)
 {
 	int fd;
+	int fd_sample = 0;
 	int ret;
-	ssize_t len;
 	off_t size;
 	int i;
-	uint16_t crc;
 	struct stat st;
-	int data_len = bufsize - sizeof(g_buf.crc);
+	uint16_t crc1;
+	uint16_t crc2;
+	uint16_t data_len;
 	
 	fd = open(path, O_RDWR);
 	if (fd < 0) {
 		if (errno == ENOENT) {
-			printf("Verify_file File is not exist, no need to verify!! path : %s\n", path);
+			printf("File to be verified does not exist, no need to verify!! path : %s\n", path);
 			*index = FILE_CHUNK_NO_INDEX;
 			return OK;
 		}
 		
-		printf("verify_file %s open error... errno : %d\n", path, errno);
+		printf("Unable to open file : %s for verifying, errno : %d\n", path, errno);
 		return ERROR;
 	}
 
+	data_len = bufsize - sizeof(g_buf.crc);
```

Comment:
now this can be removed, only used in line 302..

Reply:
Done